docs/integrations/containers-orchestration/rabbitmq.md

```diff
@@ -551,8 +402,13 @@ Use this dashboard to:
 <img src={useBaseUrl('img/integrations/containers-orchestration/RabbitMQ-Logs.png')} alt="RabbitMQ dashboards" />
 
 
+## Create monitors for RabbitMQ app
+
+import CreateMonitors from '../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
 
-## RabbitMQ Alerts
+### RabbitMQ Alerts
 
 Sumo Logic provides out-of-the-box alerts available via [Sumo Logic monitors](/docs/alerts/monitors). These alerts are built based on logs and metrics datasets and have preset thresholds based on industry best practices and recommendations.
```

Comment:
The content under this heading should be removed and we should only have alert list here.